api/services/S3SiteRemover.js

```diff
@@ -1,55 +1,62 @@
-const AWS = require("aws-sdk")
-const config = require("../../config")
+const AWS = require('aws-sdk');
+const config = require('../../config');
 
-const s3Config = config.s3
+const s3Config = config.s3;
 const s3Client = new AWS.S3({
   accessKeyId: s3Config.accessKeyId,
   secretAccessKey: s3Config.secretAccessKey,
   region: s3Config.region,
-})
+});
 
-const removeSite = site => {
-  return Promise.all([
-    getObjectsWithPrefix(`site/${site.owner}/${site.repository}`),
-    getObjectsWithPrefix(`demo/${site.owner}/${site.repository}`),
-    getObjectsWithPrefix(`preview/${site.owner}/${site.repository}`),
-  ]).then(objects => {
-    objects = [].concat.apply([], objects)
-    return deleteObjects(objects)
-  })
-}
+const deleteObjects = (objects) => {
```

Comment:
explanatory comment about the 1000-object limit and how this works would be nice